pkgs/by-name/gh/ghostty/linux.nix

```diff
@@ -113,15 +105,14 @@ stdenv.mkDerivation (finalAttrs: {
       "-Drenderer=${renderer}"
     ]
     ++ lib.mapAttrsToList (name: package: "-fsys=${name} --search-prefix ${lib.getLib package}") {
-      inherit glslang;
+      inherit fontconfig glslang;
```

Comment:
Why is this added? Shouldn't this already be implicitly enabled by default [here](https://github.com/ghostty-org/ghostty/blob/3f7c3afaf947280bd2852626ff4599c02d9fb07e/build.zig#L300-L316) like the other deps?

Reply:
Whoops, it should.